src/main/java/uk/gov/companieshouse/insolvency/delta/processor/InsolvencyDeltaProcessor.java

```diff
@@ -41,26 +52,44 @@ public InsolvencyDeltaProcessor(InsolvencyDeltaProducer deltaProducer,
     public void processDelta(Message<ChsDelta> chsDelta) {
         try {
             MessageHeaders headers = chsDelta.getHeaders();
+            final ChsDelta payload = chsDelta.getPayload();
+            final String logContext = payload.getContextId();
+            final Map<String, Object> logMap = new HashMap<>();
             final String receivedTopic =
                     Objects.requireNonNull(headers.get(KafkaHeaders.RECEIVED_TOPIC)).toString();
-            final ChsDelta payload = chsDelta.getPayload();
+
 
             ObjectMapper mapper = new ObjectMapper();
             InsolvencyDelta insolvencyDelta = mapper.readValue(payload.getData(),
                     InsolvencyDelta.class);
 
-            logger.trace(String.format("DSND-362: InsolvencyDelta extracted "
-                    + "from a Kafka message: %s", insolvencyDelta));
-
             /** We always receive only one insolvency/charge per delta in a list,
              * so we only take the first element
              * CHIPS is not able to send more than one insolvency per delta.
              **/
+
             Insolvency insolvency = insolvencyDelta.getInsolvency().get(0);
             InternalCompanyInsolvency internalCompanyInsolvency = transformer.transform(insolvency);
+            final String companyNumber = insolvency.getCompanyNumber();
+
+            logger.trace(String.format("DSND-362: InsolvencyDelta extracted "
+                    + "from a Kafka message: %s", insolvencyDelta));
```

Comment:
This log should be where it was before just after the delta - if transform fails we won't be able to see the delta and will lead us down the wrong path

Reply:
Addressed.